llvm/include/llvm/Target/GlobalISel/Combine.td

```diff
@@ -1156,6 +1156,14 @@ def udiv_by_pow2 : GICombineRule<
 def intdiv_combines : GICombineGroup<[udiv_by_const, sdiv_by_const,
                                       sdiv_by_pow2, udiv_by_pow2]>;
 
+def urem_by_const : GICombineRule<
+  (defs root:$root),
+  (match (wip_match_opcode G_UREM):$root,
```

Comment:
We want to move away from using wip_match_opcode, can you use the new syntax that lists opcodes?